spec/features/job_seekers_can_view_vacancies_spec.rb

```diff
@@ -125,6 +125,27 @@
     expect(page).not_to have_content(I18n.t('jobs.no_jobs'))
   end
 
+  scenario 'The search button text changes from \'Search\' to \'Refine search\' when the filters are applied' do
+    Vacancy.__elasticsearch__.client.indices.flush
+    visit jobs_path
+
+    expect(find('.button').value).to eq(I18n.t('buttons.apply_filters'))
+  end
+
+  scenario 'Should have button with text \'Refine search\' if search criteria are applied' do
+    visit jobs_path
+
```

Comment:
Can remove lines 133-137. Then the one scenario verifies the existence of both `Search` and `Refine Search` after applying the criteria.